Traceback (most recent call last):
  File "c:\users\au614889\anaconda3\envs\anomaly_detection\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\au614889\anaconda3\envs\anomaly_detection\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "c:\users\au614889\anaconda3\envs\anomaly_detection\lib\site-packages\guild\op_main.py", line 33, in <module>
    from guild import op_util
  File "c:\users\au614889\anaconda3\envs\anomaly_detection\lib\site-packages\guild\op_util.py", line 34, in <module>
    from guild import guildfile
  File "c:\users\au614889\anaconda3\envs\anomaly_detection\lib\site-packages\guild\guildfile.py", line 29, in <module>
    from guild import opref
  File "c:\users\au614889\anaconda3\envs\anomaly_detection\lib\site-packages\guild\opref.py", line 23, in <module>
    from guild import util
  File "c:\users\au614889\anaconda3\envs\anomaly_detection\lib\site-packages\guild\util.py", line 25, in <module>
    import shutil
  File "c:\users\au614889\anaconda3\envs\anomaly_detection\lib\shutil.py", line 41, in <module>
    from grp import getgrnam
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 914, in _find_spec
  File "<frozen importlib._bootstrap_external>", line 1342, in find_spec
  File "<frozen importlib._bootstrap_external>", line 1314, in _get_spec
  File "<frozen importlib._bootstrap_external>", line 1439, in find_spec
  File "<frozen importlib._bootstrap_external>", line 87, in _path_stat
KeyboardInterrupt
